src/vis/scatter/utils.ts

```diff
@@ -208,11 +233,15 @@ export async function createScatterTraces(
                 },
                 symbol: shapeCol ? shapeCol.resolvedValues.map((v) => shapeScale(v.val as string)) : 'circle',
                 color: colorCol
-                  ? colorCol.resolvedValues.map((v) =>
-                      selected[v.id] ? '#E29609' : colorCol.type === EColumnTypes.NUMERICAL ? numericalColorScale(v.val as number) : scales.color(v.val),
-                    )
+                  ? hasSelected
+                    ? colorCol.resolvedValues.map((v) =>
+                        selected[v.id] ? (colorCol.type === EColumnTypes.NUMERICAL ? numericalColorScale(v.val as number) : scales.color(v.val)) : '#2e2e2e',
+                      )
+                    : colorCol.resolvedValues.map((v) =>
+                        colorCol.type === EColumnTypes.NUMERICAL ? numericalColorScale(v.val as number) : scales.color(v.val),
+                      )
                   : xCurr.resolvedValues.map((v) => (selected[v.id] ? '#E29609' : '#2e2e2e')),
```

Comment:
Should we extract the repeating color values to some config/constant file? Adding some variable names would also provide some documentation.

Reply:
Yes, agreed. Extracted them to a constants.ts file and changed it here as well as in the strip plots